src/core/vectortile/qgsvectortilelayerrenderer.cpp

```diff
@@ -278,9 +280,14 @@ void QgsVectorTileLayerRenderer::decodeAndDrawTile( const QgsVectorTileRawData &
     ctx.painter()->setPen( pen );
     ctx.painter()->setBrush( brush );
     ctx.painter()->drawPolygon( tile.tilePolygon() );
-#if 0
-    ctx.painter()->setBrush( QBrush( QColor( 255, 0, 0 ) ) );
-    ctx.painter()->drawText( tile.tilePolygon().boundingRect().center(), tile.id().toString() );
+#if 1
```

Comment:
```suggestion
#if 0
```

Reply:
I'd rather leave this -- it's only shown when the tile boundaries are visible, and that option is for debugging only and isn't exposed anywhere in the application.